static/js/components/PersonalForm.js

```diff
@@ -74,34 +79,17 @@ export default class PersonalForm extends ProfileFormFields {
               {...this.defaultInputComponentProps()}
             />
           </Cell>
+          <Cell col={12}>
+            {this.boundGeosuggest(addressMapping, "Current address",
+              {
+                id: "current-home",
+                placeholder: "Example: 100 Main Street, Anytown, 01234, United States",
+                types: ["geocode"]
```

Comment:
Small comment: I think this should be `address` instead of `geocode`. Reason being we need to be sure that we are guaranteed a precise street address. I'm fairly certain in my early testing that `geocode` accepted addresses like "Boston, MA". Worth verifying again though.